Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Compositor/ShaderManager.cs

```diff
@@ -0,0 +1,345 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#define OUTPUT_YUV
+
+using UnityEngine;
+using System.Collections;
+using System.Runtime.InteropServices;
+using System;
+using UnityEngine.Rendering;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.Compositor
+{
+    public class ShaderManager : MonoBehaviour
+    {
+#if UNITY_EDITOR
+        public CompositionManager Compositor { get; set; }
+
+        // the color image texture coming from the camera, converted to RGB. The Unity camera is "cleared" to this texture
+        public RenderTexture colorRGBTexture { get; private set; }
```

Comment:
nit: these comments should be changed to three bar summaries. Otherwise this won't resolve with intelisense/documentation generation.